specs/subresourceintegrity/spec.markdown

```diff
@@ -659,10 +659,10 @@ Replace step 14.1 of HTML5's ["prepare a script" algorithm][prepare] with:
 ## Proxies
 
 Optimizing proxies and other intermediate servers which modify the
-content of fetched resources MUST ensure that the digest associated
-with those resources stays in sync with the new content. One option
+response MUST ensure that the digest associated
```

Comment:
The pluralization doesn't match up here. Probably should read "...which modify the responses MUST ensure that..." (note added "s" to "response").